### PR TITLE
implemented remove condition

### PR DESCRIPTION
**Pull request by guergana**
+ quiet button will be added when new version of wikit is released
+ added removeCondition action and mutation
+ added tests

### CODE REVIEW

File: src/components/QueryCondition.vue [hunk 1]
```diff
@@ -19,12 +19,19 @@
 			:placeholder="$i18n('query-builder-input-value-placeholder')"
 			:disabled="isTextInputDisabled()"
 		/>
+		<Button
```
Comment (micgro42) [inline src/components/QueryCondition.vue:22]: This should probably be its own simple subcomponent as well, similar to ValueTypeDropdown and PropertyLookup. The idea is to try to maintain "one level of abstraction" within the template as well.

But that could also be done in a follow-up pull request

File: tests/unit/store/mutations.spec.ts [hunk 2]
```diff
@@ -97,6 +97,37 @@ describe( 'mutations', () => {
...
 
+	it( 'removeCondition', () => {
+		const keptRow = {
+			valueData: { value: 'foo', valueError: null },
+			propertyData: { id: 'P123', label: 'abc', datatype: 'string', propertyError: null },
+			propertyValueRelationData: { value: PropertyValueRelation.Matching },
+			conditionId: '0.123',
+		};
+		const state: RootState = {
+			conditionRows: [ {
```
Comment (micgro42) [inline tests/unit/store/mutations.spec.ts]: You can replace this first object with the `keptRow` variable from above :)

**Review by micgro42 — COMMENTED** [review]
Thank you for taking this on! We should extract the component, but that should be relatively

File: src/components/QueryCondition.vue (hunk 1 above)
Comment (guergana) [reply src/components/QueryCondition.vue:22]: Let's do it in a followup. :)

Comment (micgro42) [reply src/components/QueryCondition.vue:22]: :+1:

**Review by micgro42 — APPROVED** [review]

**Pull request merged**